data/human/post-war news.txt

```diff
@@ -0,0 +1,647 @@
+		"paranoia: media creators are aliens"
+		"paranoia: people in my life are aliens"
+		"paranoia: people in power are aliens"
+
+phrase "somebody suspects that"
+	word
+		"I suspect that"
+		"I'm positive that"
+		"People say"
+		"People say that"
+		"They say"
+		"I hear rumors that"
+		"I wouldn't be surprised if"
+		"I can't help feeling like"
+		"There's a good chance that"
+		"It's definitely possible that"
+		"I'm pretty sure"
+		"Everyone I know is certain that"
+		"My crazy uncle is always jabbering on about how"
+		"My crazy mother-in-law is always jabbering on about how"
+		"My last girlfriend wouldn't stop ranting about how she thinks"
+		"My last boyfriend wouldn't stop ranting about how he thinks"
+		"I can't believe how some people could think that"
+		"You'd have to be a real dope to believe that"
+		"I find myself believing that"
+		"Nobody believes me when I tell them about how"
+		"I think"
+		"I really do think"
+		"I believe that"
+		"I really do believe that"
+		"I'm sure that"
+		"Everyone knows"
+		"Everyone knows that"
+		"Everybody knows"
+		"Everybody knows that"
+		"I wish more people realized"
+		"I wish more people realized that"
+		"My nutty cousin claims that"
+		"The voices in my head say"
+		"My invisible friend knows that"
+
+phrase "paranoia: people in power are aliens"
+	phrase
+		"somebody suspects that"
+	word
+		" "
+	word
+		"the politicians"
+		"government officials"
+		"members of Parliament"
+		"people on the Paradise worlds"
+		"people on the rich worlds"
+		"those weird folks from the Deep"
+		"corporate executives"
+		"the Republic's intelligence operatives"
+		"bank managers"
+		"Syndicate executives"
+		"most planetary governors"
+		"the executives of the Tarazed corporation"
+		"the executives of the Lionheart corporation"
+		"the executives of the Megaparsec corporation"
+		"the executives of the Lovelace corporation"
+	word
+		" are actually aliens."
+		" are aliens in disguise."
+		" are just pretending to be human."
+		" are aliens."
+
+phrase "paranoia: band members are aliens"
+	word
+		"I watched the video of"
+		"I listened to"
+		"I just got tickets to see"
+		"I just got back from seeing"
+		"I'm just coming back from seeing"
+		"I just heard"
+		"I just saw"
+		"I had a backstage pass to"
+	word
+		" "
+	phrase
+		"band"
+	word
+		" "
+	word
+		"playing a concert"
+		"doing a show"
+		"in concert"
+		"at their concert"
+		"playing a benefit concert"
+		"live"
+		"on tour"
+		"at a live concert"
+		"playing a gig"
+		"live in concert"
+	word
+		" at Tarazed Stadium"
+		" at the amphitheater"
+		" at a concert hall"
+		" in a nightclub"
+		" at Betelgeuse Stadium"
+		" at Lionheart Arena"
+		" at the Grange"
+		" at a resort"
+		" at Canyonland Park"
+		" in a stylin' party-hall"
+		""
+	word
+		", but after what I heard"
+		", but after what I've seen"
+		", and now that I've really looked at them"
+	word
+		" I'm sure they're"
+		" I know they're"
+		" it's clear to me now they're"
```

Comment:
```suggestion
		" it's clear to me that they're"
```
`, and now that I've really looked at them it's clear to me now they're`: don't like the possibility for the two `now`s close to each other.